src/ui/etherInput.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class='eth-input'>
+    <input id='eth-value'
+      type='number' step='0.0001' min='0'
+           :value="value" @change="$emit('update:value', $event.target.value)"
+      />
+    <p class='coin'>ETH</p>
+    <p class='text'>
+      <i>That's roughly ${{ (value * exchangeRate).toFixed(2) }}</i>
```

Comment:
This is so simple I love it

Reply:
Agreed, this is an awesome feature that will make our site much more understandable.